src/content/en/fundamentals/media/mobile-web-video-playback.md

```diff
@@ -502,6 +502,26 @@ resources in an infinite scroll case.
           type="video/mp4">
 </video>
 
+### Play only one video at a time
+
+If there are more than one video on a page, I would suggest you only play one
+and pause the other ones automatically so that user doesn't have to hear
+multiples audio tracks playing simultaneously.
+
+    var videos = Array.from(document.querySelectorAll('video'));
```

Comment:
Maybe at least it should be noted that this call should happen after the videos are added?